build.gradle

```diff
@@ -266,7 +266,7 @@ subprojects {
   jar {
     manifest {
       attributes 'Implementation-Title': project.name,
-                 'Implementation-Version': version,
+                 'Implementation-Version': archiveVersion,
```

Comment:
does this work with the 5.x version of gradle?

Reply:
Oh, I haven't checked that. Would it matter? For building the repo, we can make Gradle 6 required, no?

Reply:
oh right, oops!